src/nunavut/lang/cpp/__init__.py

```diff
@@ -549,9 +545,53 @@ def filter_includes(language: Language, t: pydsdl.CompositeType, sort: bool = Tr
     :param bool sort: If true the returned list will be sorted.
     :return: a list of include headers needed for a given type.
     """
+    return IncludeGenerator(language, t).generate_include_filepart_list(language.extension, sort)
+
+
+@template_language_filter(__name__)
+def filter_destructor_name(language: Language, instance: pydsdl.Any) -> str:
+    """
+    Returns a token that is the local destructor name. For example:
+
+    .. invisible-code-block: python
 
-    include_gen = IncludeGenerator(language, t, filter_id, filter_short_reference_name)
-    return include_gen.generate_include_filepart_list(language.extension, sort)
+        from nunavut.lang.cpp import filter_destructor_name
+        from unittest.mock import MagicMock
+        import pydsdl
+
+        my_type = MagicMock(spec=pydsdl.ArrayType)
+        my_type.version = MagicMock()
```

Comment:
I don't follow why is this necessary. Copy-paste error? The two references below also seem out of place.

Reply:
Sorry, I'm not getting your comment here. Can you clarify?

Reply:
Neither `ArrayType` nor any of its descendants define an attribute named `version`, which means that the following statements are incorrect and should be removed:

> `my_type.version = MagicMock()`
> `my_type.version.major = 1`
> `my_type.version.minor = 0`